### ISSUE TITLE
Not sure a nextjs bug but img src are being altered in production.

### ISSUE THREAD

**Issue by SaudTauqeer** [zeit/next.js#9067]
# Next.js altering request url's only after deployment.
I have something weird going on and I cannot wrap my head around it.
I am using create-next-app and I have a simple `.map()` the function that generates out URLs.
These URLs point to images hosted on s3, which is a public s3 bucket. 
Now, the requests work fine both in local development and production build.
But when I host the exact same setup the error starts to occur:
expected request: `https://draftxr.s3.us-east-2.amazonaws.com/1234/f51594bb-4bec-436d-92a2-f5ba4949c8bb.png`
error request: `https://app.draftxr.com/vr/draftxr.s3.us-east-2.amazonaws.com/1234/f51594bb-4bec-436d-92a2-f5ba4949c8bb.png`
In the error request `https://app.draftxr.com` is the domain name and `/vr` is the component which is sending the request.
# What I have tried so far.
- Tried hosting it on aws (current hosting), Now and Heroku.
- Tried rendering the app with SSR disabled.
Again, the same code and setup works exactly as expected in local production build and development build but this issue only happens in when hosting the production build.

**Comment by timneutkens** [comment]
Please follow the issue template.

https://github.com/zeit/next.js/issues/new/choose

https://github.com/zeit/next.js/issues/new?template=8.Question_about_next.md

```
# Question about Next.js

GitHub Issues are reserved for Bug reports and Feature requests. The best place to get your question answered is to post it on https://spectrum.chat/next-js.
```

**Comment by balazsorban44** [comment]
This issue has been automatically locked due to no recent activity. If you are running into a similar issue, please create a new issue with the steps to reproduce. Thank you.
